### DISCUSSION TITLE
Pure/Constant Interpolated Strings

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/csharplang#3267] [Language Ideas]
It would be great if I could do this:
````
const string V1 = "test";
const double V2 = 123.45678;
const string V3 = #"I am {V1}ing this out {V2:0.00} times.";
````

My understanding is that currently $"" does some culture-specific stuff.  It would be great if # could be used to generate const-ish strings devoid of any culture applications (invariant culture?)

**Comment by HaloFour** [comment]
#2951

**Comment by canton7** [comment]
Also #384/#2077 and #177/#3044
